manifests/cloud-platform-alt-dns/coredns-corefile.tmpl

```diff
@@ -0,0 +1,24 @@
+. {
```

Comment:
A couple of thoughts on this:

- I don't think the cluster will complete installation without the ingress record. 
- Since it's likely this will need to support dual stack at some point it might not hurt to just include the [logic for the secondary record](https://github.com/openshift/machine-config-operator/blob/f41e20d6ba3951ac9ce9411f0511558c34e60e53/templates/common/on-prem/files/coredns-corefile.yaml#L21) now.

Reply:
> A couple of thoughts on this:
> 
> * I don't think the cluster will complete installation without the ingress record.

For this feature, we are lucky to be able to do this in stages. In the current stage, we continue to allow the Ingress operator to start the LB service that creates the Ingress LBs and configures the cloud DNS for *.apps. 
The in-cluster CoreDNS pod only has to be able to resolve api and api-int URLs. In the next stage, we instruct the Ingress operator to only configure the LBs and skip the configuration of the cloud DNS. We use the ingress LB IPs to update the CoreDNS pod to resolve *.apps and  This gives us the ability to test our wok in smaller increments.

Reply:
@cybertron I think both your concerns have been addressed.